app/packs/src/utilities/selectHelper.js

```diff
@@ -0,0 +1,26 @@
+// Format data for react-select component when using with users data as returned by the API
+// { users: [{ id: 1, name: 'John Doe', initials: 'JD', type: 'Person' }, ...]}
+
+const filterLabel = (user, withType = false) => (
+  withType ? `${user.name} (${user.initials} - ${user.type})` : `${user.name} (${user.initials})`
+);
+
+const filterData = (data, currentUserId = null) => {
```

Comment:
maybe rename method to something that makes more clear what it does like: getUserFromData